packages/terra-avatar/src/common/Avatar.module.scss

```diff
@@ -63,7 +63,9 @@
 
   .avatar > .image {
     border-radius: 50%;
+    font-family: 'object-fit: cover;'; //This is added to provide settings for object-fit-images polyfill
     height: 100%;
+    object-fit: cover;
```

Comment:
if we use object-fit, is it necessary to define 100% height and width?

Reply:
What I have found is height and width 100% is required for non-square images.